.github/workflows/build.yml

```diff
@@ -0,0 +1,82 @@
+name: "Build workflow"
+on:
+  pull_request:
+  push:
+    tags:
+      - v**
+    branches:
+      - main
+
+env:
+  PYTHON_VERSION: "3.10" # Duplicated in `build-test` due to Workflow limitations.
+  SANDBOX_CONFIG: dev
+  SANDBOX_GENESIS: genesis/dev/genesis.json
+
+jobs:
+  build-test:
+    runs-on: ubuntu-20.04
+    container: python:${{ matrix.python }}
+    strategy:
+      matrix:
+        python: [ "3.8", "3.9", "3.10" ]
+    steps:
+      - run: python3 --version
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: pip, lint, and units
+        run: make build-and-test
+  run-integration-tests:
+    runs-on: ubuntu-20.04
+    # container: python:${{ matrix.python }}
+    strategy:
+      matrix:
+        python: [ "3.8", "3.9", "3.10" ]
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v3
+        with:
+          python-version: "${{ matrix.python }}"
+          # python-version: "${{ env.PYTHON_VERSION }}"
+      - name: Test Python version
+        # expected="${{ env.PYTHON_VERSION }}"
```

Comment:
@tzaffi Minor:  Friendly pointer to commented out code.

```suggestion
```

Reply:
👍